lib/init.coffee

```diff
@@ -25,9 +34,11 @@ module.exports =
         return new Promise (resolve, reject) =>
           filePath = textEditor.getPath()
           json = []
+          hints = (('--hint=' + h) for h in @hlintHints)
           process = new BufferedProcess
             command: @executablePath
-            args: [filePath, '--json', '--hint=Default', '--hint=Dollar', '--hint=Generalise']
+            args: [filePath, '--json'].concat hints 
+            # args: [filePath, '--json', '--hint=Default', '--hint=Dollar', '--hint=Generalise']
             # args: [filePath, '--json']
```

Comment:
Done!

On Tue, Apr 5, 2016 at 4:34 PM, Landon Abney <notifications@github.com>
wrote:

> In lib/init.coffee
> <https://github.com/AtomLinter/linter-hlint/pull/21#discussion_r58631798>:
>
> >            process = new BufferedProcess
> >              command: @executablePath
> > -            args: [filePath, '--json', '--hint=Default', '--hint=Dollar', '--hint=Generalise']
> > +            args: [filePath, '--json'].concat hints
> > +            # args: [filePath, '--json', '--hint=Default', '--hint=Dollar', '--hint=Generalise']
> >              # args: [filePath, '--json']
>
> Might as well cut this line out as it seems like duplication to me.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/AtomLinter/linter-hlint/pull/21/files/d93914e35fa765edc56e4f8a3b43e6af6e42edba#r58631798>
>